### PR TITLE
- reduced interface of SystemCmd

### PR DESCRIPTION
**Pull request by aschnell**
The PR reduces the interface of SystemCmd to what is currently used. It also unifies the behaviour of the constructors (one even uses the other).

### CODE REVIEW

File: storage/Utils/SystemCmd.h [hunk 1]
```diff
@@ -43,47 +42,79 @@ namespace storage
...
+
+	    /**
+	     * The command to be executed.
+	     */
+	    string command;
+
+	    /**
+	     * Should exceptions be thrown or not?
+	     */
+	    ThrowBehaviour throw_behaviour;
```
Comment (shundhammer) [inline storage/Utils/SystemCmd.h]: Please add a constructor that properly initializes those member variables that don't have a reasonable default constructor - like the ThrowBehaviour.

Comment (aschnell) [inline storage/Utils/SystemCmd.h]: The existing constructor does that. Do you additionally want a default constructor?

Comment (shundhammer) [inline storage/Utils/SystemCmd.h]: Argh. Forget it; I overlooked that one.

**Comment by shundhammer** [comment]
LGTM

**Pull request merged**